### DISCUSSION TITLE
Alarm "DelayedOn": Make the delay until arming configurable

### DISCUSSION THREAD

**Discussion by sebastian-enz** [JoDehli/PyLoxone#314] [Ideas]
According to the Loxone docs, the arming delay can be passed as a parameter "Da" (see: https://www.loxone.com/enen/wp-content/uploads/sites/3/2016/10/EN_KB_Diagram_Config8_API_Structure.pdf Page 15). With the recent pull request #309, the delay is not configurable. However, me and some other people would probably want to set a custom delay such as 0 seconds, and not use the default 60 seconds.

**Comment by zach-ovic** [comment]
Hi, I'm the author of the pull-request. I'll take a look into it. But my first impression is that "Da" parameter they are talking about is a parameter in Loxon Config. So you can now configure it by creating Virtual input, which you connect to Da input on Alarm block in Loxone config. There is no way to set the delay using WebSocket command it self.

By the way, you are looking into documentation from 2016. Current documentation is [https://www.loxone.com/wp-content/uploads/datasheets/StructureFile.pdf](here).
